re_data_ui/src/pages/Alerts.tsx

```diff
@@ -0,0 +1,141 @@
+                    >
+                      Model
+                    </th>
+                    <th
+                      scope="col"
+                      className="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+                    >
+                      Message
+                    </th>
+                    <th
+                      scope="col"
+                      className="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+                    >
+                      Metric Value
+                    </th>
+                    <th
+                      scope="col"
+                      className="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+                    >
+                      Time Window
+                    </th>
+                    <th scope="col" className="relative px-6 py-3">
+                      <span className="sr-only">Details</span>
+                    </th>
+                  </tr>
+                </thead>
+                <tbody className="bg-white divide-y divide-gray-200">
+                  {anomalies.map((anomaly) => (
+                    <tr key={anomaly.id}>
+                      <td className="px-6 py-4 whitespace-nowrap">
+                        <span
+                          className="badge mb-3 bg-red-600 rounded-full px-2.5 py-1 text-center object-right-top text-white text-sm mr-3"
+                        >
+                          !
+                        </span>
+                        <span
+                          className="text-sm text-gray-900"
+                        >
+                          {stripQuotes(anomaly.table_name)}
+                        </span>
+                      </td>
+                      <td className="px-6 text-sm py-4 whitespace-nowrap">
+                        <div
+                          className="text-gray-900"
+                        >
+                          {generateAlertMessage(anomaly)}
+                        </div>
+                      </td>
+                      <td className="px-6 py-4 text-sm whitespace-nowrap">
+                        {generateAnomalyValue(anomaly)}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-sm text-gray-500">
+                        {dayjs(anomaly.time_window_end).format(dateTimeFormat)}
+                      </td>
+                      <td className="px-6 py-4 whitespace-nowrap text-right text-sm font-medium">
+                        <Link
+                          to={`/graph?model=${stripQuotes(anomaly.table_name)}`}
+                          className="text-indigo-600 hover:text-indigo-900"
+                        >
+                          Details
+                        </Link>
+                      </td>
+                    </tr>
+                  ))}
+
+                  {schemaChanges.map((change) => (
```

Comment:
for this, will probably need to have a list containing both schema changes & anomalies. Actually this is something I consider changing dbt-operation for